torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -704,9 +787,12 @@ def create_from_config(cls, config: CppMicroGemmConfig):
                 continue
             if (
                 config.input_dtype == input_dtype
-                and config.output_dtype == output_dtype
                 and config.compute_dtype == compute_dtype
                 and config.input2_dtype == input2_dtype
+                and config.output_dtype == output_dtype
+                # For int8 WoQ GEMM, output dtype would be same as that of the activation,
+                # but the micro-kernels computes output in Float, which is converted to BF16
+                # in the GEMM template.
```

Comment:
This is same for bf16/fp16. I don't think we need additional comments here. No need to change the code here?

Reply:
I added these comments here because it may not be immediately clear to someone looking at the code for the first time that the `output_dtype` to be passed here doesn't need to be the _actual_ dtype of the output.